lms/extractors/textfile.py

```diff
@@ -8,17 +8,16 @@
     {7, 8, 9, 10, 12, 13, 27}
     | set(range(0x20, 0x100)) - {0x7f},
 ))
+ALLOWED_EXTENSIONS = {'css', 'html', 'js', 'py', 'sql'}
```

Comment:
WDYT about using `lms.utils.files` for that?

Reply:
Good idea